go/common/config.go

```diff
@@ -113,26 +114,42 @@ type StorageConfig struct {
 	Code    StoreString `json:"code"`
 }
 
+// One unified limits struct for both worker defaults and per-lambda overrides.
+// For per-lambda ol.yaml, zero values mean "use worker defaults".
 type LimitsConfig struct {
-	// how many processes can be created within a Sandbox?
-	Procs int `json:"procs"`
-
-	// how much memory can a regular lambda use?  The lambda can
-	// always set a lower limit for itself.
-	Mem_mb int `json:"mem_mb"`
+	// process & memory / CPU controls
+	Procs       int `json:"procs" yaml:"procs"`
+	Mem_mb      int `json:"mem_mb" yaml:"mem_mb"`
+	CPU_percent int `json:"cpu_percent" yaml:"cpu_percent"`
+	Swappiness  int `json:"swappiness" yaml:"swappiness"`
 
-	// what percent of a core can it use per period?  (0-100, or more for multiple cores)
-	CPU_percent int `json:"cpu_percent"`
+	// worker default for runtime cap (seconds). Per-lambda may set runtime_sec.
+	Max_runtime_default int `json:"max_runtime_default" yaml:"max_runtime_default"`
```

Comment:
Do we need a default runtime?  I thought we would have a different struct instance for defaults, not a different field within the struct?

Is there a reason we would have a regular and default for runtime, but not for other fields like procs?